### PR TITLE
Update CHANGELOG.md for the 202011.00 release.

### PR DESCRIPTION
**Pull request by sarenameas**
The title says it all.

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

**Review by leegeth — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+# Changelog for AWS IoT Device SDK for Embedded C
+
+## 202011.00 (November 2020)
+
+### Major Changes
+
+- This release introduces the re-factored **HTTP** client library, **AWS IoT Device Defender** service client library, and **AWS IoT Jobs** service library.
+These libraries have gone through code quality checks including verification that no function has a [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html) score greater than 8, and checks against deviations from the mandatory rules in the [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx). These libraries have also undergone both static code analysis from [Coverity statical analysis](https://scan.coverity.com/) and validation of memory safety through the [CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721).
+
+- This release uses submodule references to the following new individual repositories for the HTTP, AWS IoT Defender service client, and AWS IoT Jobs service libraries under the `libraries` folder:
```
Comment (aggarw13) [inline CHANGELOG.md:10]:
```suggestion
- This release uses submodule references to the following new individual repositories for the HTTP, AWS IoT Device Defender service, and AWS IoT Jobs service client libraries under the `libraries` folder:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+   - [FreeRTOS/coreHTTP](https://github.com/FreeRTOS/coreHTTP) for the HTTP client library
+   - [aws/device-defender-for-aws-iot-embedded-sdk](https://github.com/aws/device-defender-for-aws-iot-embedded-sdk) for the AWS IoT Device Defender service client library.
```
Comment (aggarw13) [inline CHANGELOG.md:12]:
```suggestion
   - [aws/device-defender-for-aws-iot-embedded-sdk](https://github.com/aws/device-defender-for-aws-iot-embedded-sdk) for the AWS IoT Device Defender client library.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,28 @@
-# Change Log for AWS IoT Device SDK for Embedded C
+# Changelog for AWS IoT Device SDK for Embedded C
+
+## 202011.00 (November 2020)
+
+### Major Changes
+
+- This release introduces the re-factored **HTTP** client library, **AWS IoT Device Defender** service client library, and **AWS IoT Jobs** service library.
```
Comment (aggarw13) [inline CHANGELOG.md:7]:
```suggestion
- This release adds the re-factored **HTTP** client library, **AWS IoT Device Defender** service client library, and **AWS IoT Jobs** service library to the list of supported C-SDK libraries.
```

File: CHANGELOG.md (hunk 1 above)
Comment (sarenameas) [reply CHANGELOG.md:10]: Not sure if client should follow jobs

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,28 @@
...
+These libraries have gone through code quality checks including verification that no function has a [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html) score greater than 8, and checks against deviations from the mandatory rules in the [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx). These libraries have also undergone both static code analysis from [Coverity statical analysis](https://scan.coverity.com/) and validation of memory safety through the [CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721).
```
Comment (tanmoysen) [inline CHANGELOG.md:8]: Do we need to post the CBMC video? Or should we use the link: http://www.cs.cmu.edu/~modelcheck/cbmc/

Comment (sarenameas) [reply CHANGELOG.md:8]: We can use that link instead

**Review by leegeth — APPROVED** [review]

**Review by tanmoysen — APPROVED** [review]

**Review by abhidixi11 — APPROVED** [review]

**Pull request merged**